### PR TITLE
First draft of CSS Exercises

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: src/chapters/css/exercises.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Exercises: CSS
```
Comment (jimflores5) [inline src/chapters/css/exercises.rst]: Cool exercises, Sally.  I only found one grammar edit.

File: src/chapters/css/exercises.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+==============
+
+We have built a website for you to test your CSS knowledge on. Check out the `repl.it <https://repl.it/@launchcode/CSSExercises/>`_!
+
+For the exercises, add the following style rules to ``style.css``:
```
Comment (jimflores5) [inline src/chapters/css/exercises.rst]: There are multiple tasks for styling elements (``p``, ``h1``, etc.) and one for styling an id, but there are no direct examples for styling a *class*. Students could create a ``centered`` class, for example, and apply that inside the tags, or they could add ``text-align: center;`` to each element in the CSS.

I would vote for adding another ``id`` example and a couple of ``class`` examples in the exercises, especially if they can be structured to reinforce the priority for element/class/id styling.

File: src/chapters/css/exercises.rst [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+1. Change the background color to yellow.
+2. Change all paragraph text color to red.
```
Comment (jimflores5) [inline src/chapters/css/exercises.rst]: OUCH! Red text on a yellow background. My eyes are hurting just thinking about this. ;-)

File: src/chapters/css/exercises.rst [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+3. Change all ``h1`` to 36 px font size.
+4. Align all text to the center of the page.
```
Comment (jimflores5) [inline src/chapters/css/exercises.rst]: What do you think about making this step have multiple parts?  I can think of at least three ways to center all the text on the page.  Perhaps we can ask the students to do something similar?

Example:
4. Align all text to the center of the page.
    a. Now find a *different* way to center the text on the page.
    b. Can you find *another* way to center the text on the page?

File: src/chapters/css/exercises.rst [hunk 5]
```diff
@@ -0,0 +1,13 @@
+Exercises: CSS
+==============
+
+We have built a website for you to test your CSS knowledge on. Check out the `repl.it <https://repl.it/@launchcode/CSSExercises/>`_!
+
+For the exercises, add the following style rules to ``style.css``:
+
+1. Change the background color to yellow.
+2. Change all paragraph text color to red.
+3. Change all ``h1`` to 36 px font size.
```
Comment (jimflores5) [inline src/chapters/css/exercises.rst]: How do you feel about introducing responsive text sizing as a subset of this?  I remember an exercise in LC101 where we changed the window size to see how the text responded using ``36 px`` vs. something like ``2.5vw``.

File: src/chapters/css/exercises.rst [hunk 6]
```diff
@@ -0,0 +1,13 @@
+Exercises: CSS
+==============
+
+We have built a website for you to test your CSS knowledge on. Check out the `repl.it <https://repl.it/@launchcode/CSSExercises/>`_!
```
Comment (jimflores5) [inline src/chapters/css/exercises.rst]: Grammar: Remove ending preposition.

How about, 
a) "We built a website for you to test your CSS knowledge."
b) "We created a website where you can practice your CSS styling."
c) "ARRR! Use yer CSS skills ter make yonder website shipshape."
d) Other... Just don't end with a preposition.

File: src/chapters/css/exercises.rst [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+For the exercises, add the following style rules to ``style.css``:
+
+1. Change the background color to yellow.
+2. Change all paragraph text color to red.
+3. Change all ``h1`` to 36 px font size.
+4. Align all text to the center of the page.
+5. Change the font color of elements with the id, ``cool-text``, to blue. 
+
```
Comment (jimflores5) [inline src/chapters/css/exercises.rst]: I noticed that there is a ``script.js`` file in the repl.it and a ``<script>`` element in the html. Were you going to have the students add some JavaScript to the practice page?

File: src/chapters/css/exercises.rst (hunk 1 above)
Comment (gildedgardenia) [reply src/chapters/css/exercises.rst]: Thanks, Jim!

File: src/chapters/css/exercises.rst (hunk 3 above)
Comment (gildedgardenia) [reply src/chapters/css/exercises.rst]: Very true! Maybe I will make it green on yellow.

File: src/chapters/css/exercises.rst (hunk 5 above)
Comment (gildedgardenia) [reply src/chapters/css/exercises.rst]: We didn't cover this in the chapter.

File: src/chapters/css/exercises.rst (hunk 7 above)
Comment (gildedgardenia) [reply src/chapters/css/exercises.rst]: Nope! Repl.it just wants it there and doesn't like it when I remove it.

File: src/chapters/css/exercises.rst (hunk 4 above)
Comment (gildedgardenia) [reply src/chapters/css/exercises.rst]: They only learn how to use ``text-align``.

Comment (jimflores5) [reply src/chapters/css/exercises.rst]: Yes, but you can ``text-align`` each tag, ``text-align`` a class, or encase the tags in one ``div`` element and ``text-align`` that.

File: src/chapters/css/exercises.rst (hunk 7 above)
Comment (jimflores5) [reply src/chapters/css/exercises.rst]: Ah. I see.

Could we add some code?  If not, maybe add a disclaimer about what the file would be used for if we wanted it.

File: src/chapters/css/exercises.rst (hunk 2 above)
Comment (gildedgardenia) [reply src/chapters/css/exercises.rst]: I think I got some good new exercises! Thanks!

File: src/chapters/css/exercises.rst (hunk 6 above)
Comment (gildedgardenia) [reply src/chapters/css/exercises.rst]: Done! Thanks!

File: src/chapters/css/exercises.rst (hunk 7 above)
Comment (gildedgardenia) [reply src/chapters/css/exercises.rst]: I add a note to it! Thanks!

File: src/chapters/css/exercises.rst (hunk 4 above)
Comment (gildedgardenia) [reply src/chapters/css/exercises.rst]: I added an exercise to use a CSS id to align only the headings to the center.

**Pull request merged**